src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,19 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
```

Comment:
Consider using flexible units for the grid-template-columns property to enhance responsiveness.